.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,12 +10,18 @@ https://github.com/intellij-rust/intellij-rust/blob/master/CONTRIBUTING.md#cla
 
 After you sign the CLA, please add your name to
 https://github.com/intellij-rust/intellij-rust/blob/master/CONTRIBUTORS.txt
-
-Also, please write a short description explaining your change in the following format: `changelog: %description%`
+Drop these lines for internal only changes
+-->
+<!--
+Also, please provide as much context as possible about the feature, attach a code sample as text, 
+a screenshot or screencast if possible in the following format: `Changelog: %description%`
```

Comment:
I'm not sure that all these things should be in changelog section.
So, I'd prefer to put "as much context as possible" in general PR description and short human-friendly description in changelog section